examples/lock-app/qpg6100/src/BoltLockManager.cpp

```diff
@@ -0,0 +1,226 @@
+/*
```

Comment:
I do wonder how much of this code could be shared with the other lock apps.  Followup, if it can be?

Reply:
Created #3840 for an overall cleanup - best to be discussed with the owners of the other examples. What's the right forum for that ?

Reply:
Either slack or just cc them on the github issue?